languages/common-lisp/concepts/sameness/about.md

```diff
@@ -1 +1,81 @@
-TODO: add information on sameness concept
+Sameness is the concept about equality of things.
+
+Common Lisp, like other languages has a set of rules on how to decide if two objects are the 'same'. These rules define four levels, each with a function that performs that level of checking. The levels are structured from strictest to loosest.
```

Comment:
Definitely a nitpick, but I think "ordered from ..." sounds a bit more natural than "structured from"

Reply:
ugh... sometimes i wonder if English really is my 'native language' :/ Good catch.